<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>179</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1,0,0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>ADC Setting</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item row="0" column="2" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="ch5RadioButton">
        <property name="text">
         <string>ch5</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="ch7RadioButton">
        <property name="text">
         <string>ch7</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="ch6RadioButton">
        <property name="text">
         <string>ch6</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="ch4RadioButton">
        <property name="text">
         <string>ch4</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="ch3RadioButton">
        <property name="text">
         <string>ch3</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="ch2RadioButton">
        <property name="text">
         <string>ch2</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="ch1RadioButton">
        <property name="text">
         <string>ch1</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="ch0RadioButton">
        <property name="text">
         <string>ch0</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1,2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Drate</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QComboBox" name="drateComboBox">
        <item>
         <property name="text">
          <string>2.5 SPS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5 SPS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 SPS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15 SPS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25 SPS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30 SPS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50 SPS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60 SPS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100 SPS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500 SPS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1000 SPS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2000 SPS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3750 SPS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7500 SPS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15000 SPS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30000 SPS</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Gain</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QComboBox" name="gainComboBox">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>64</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="adc_configPushButton">
      <property name="text">
       <string>Configure</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
